docs/podman-create.1.md

```diff
@@ -178,7 +178,7 @@ Override the key sequence for detaching a container. Format is a single characte
 
 **--device**=[]
 
-Add a host device to the container (e.g. --device=/dev/sdc:/dev/xvdc:rwm)
+Add a host device to the container. The format is `<device-on-host>:<device-on-container>:<permissions>` (e.g. --device=/dev/sdc:/dev/xvdc:rwm)
```

Comment:
I think that host device and permissions are both optional - maybe enclose them in square brackets to show that?

Reply:
Something like:
```
<devcie-on-host>[:<device-on-container>][:<permissions>]
```

Reply:
I missed the optional part, nice catch.  I'm fine with @rhatdan's suggestion with the brackets above then.  The Docker variant would work too, but I kinda prefer the shorter version.